packages/cra-template-typescript/template/gitignore

```diff
@@ -17,15 +17,7 @@
 .env.development.local
 .env.test.local
 .env.production.local
-.vscode
-.idea
 
 npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
-
-.yalc
-yalc.lock
-
-# localization files are generated, it's not source thus don't version them
-src/translations/*.json
```

Comment:
Let's not removed these values, I think we want to still ignore them

Reply:
same as above ☝️

Reply:
I reverted it